<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEditVerse</class>
 <widget class="QDialog" name="DialogEditVerse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>622</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>838</width>
    <height>622</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>838</width>
    <height>622</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pbUpdate">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>585</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbLC">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>390</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>LC</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClear">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>250</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbUC">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>290</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>UC</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>806</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Verse Info</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>35</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PRE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePre">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>103</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>35</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>POST</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePost">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>65</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Word</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLang">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lang</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leLang">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>195</y>
      <width>36</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblDict">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>200</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dict#</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDict">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>195</y>
      <width>46</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblPicture">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>200</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Picture</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leFormat">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>195</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="teWord">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>546</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSeq">
    <property name="geometry">
     <rect>
      <x>517</x>
      <y>197</y>
      <width>36</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblSeq">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>199</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Seq#</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbQuoteLeft">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>26</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Quote (pre)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbQuoteRight">
    <property name="geometry">
     <rect>
      <x>496</x>
      <y>26</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Quote (post)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbUpdate_2">
    <property name="geometry">
     <rect>
      <x>635</x>
      <y>70</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbCPP">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>340</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear PrePost</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>255</y>
     <width>126</width>
     <height>316</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOK">
   <property name="geometry">
    <rect>
     <x>689</x>
     <y>583</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbReSeq">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>435</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbMoveToPost">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>485</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>move to post</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbMoveToPre">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>530</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>move to pre</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbUpdateNClose">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>585</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Update/Close</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
